rskj-core/src/main/java/org/ethereum/util/Utils.java

```diff
@@ -254,4 +254,10 @@ public static int significantBitCount(int number) {
         }
         return result;
     }
+
+    public static String getFormattedJavaVersion(){
+        String javaVersion = System.getProperty("java.specification.version");
+        String javaMajorVersion = javaVersion.replace("1.", "");
```

Comment:
if you want we could add a check that the provided result has the format 1.x, but it is a minor thing.